src/index.js

```diff
@@ -0,0 +1,86 @@
+const fs = require("fs");
+const inquirer = require("inquirer");
+const figlet = require("figlet");
+const path = require("path");
+const open = require("open");
+const Manager = require("./lib/Manager");
+const Engineer = require("./lib/Engineer");
+const Intern = require("./lib/Intern");
+const {
+  managerQuestions,
+  confirmNextStep,
+  engineerQuestions,
+  internQuestions,
+} = require("./questions");
+const { generateHTML } = require("./generate");
+
+const init = async () => {
+  let inProgress = true;
+
+  const engineerInfo = [];
+  const internInfo = [];
+
+  const managerAnswers = await inquirer.prompt(managerQuestions);
+
+  const manager = new Manager(
+    managerAnswers.name,
+    managerAnswers.id,
+    managerAnswers.email,
+    managerAnswers.officeNumber,
+    managerAnswers.teamName
+  );
+
+  while (inProgress) {
+    const { nextStep } = await inquirer.prompt(confirmNextStep);
+
+    if (nextStep === "engineer") {
+      const engineerAnswers = await inquirer.prompt(engineerQuestions);
```

Comment:
Try to get into the habit of destructuring your objects. It gives us more visibility on what keys we have and also means we don't have to constantly reference the object.key and can just reference the key directly:

```
const { name, id, email, gitHub } = await inquirer.promot(engineerQuestions);
const engineer = new Engineer(name, id, email, github)
```
```